src/components/formio/component.tsx

```diff
@@ -16,6 +19,15 @@ export interface ComponentProps {
   children: React.ReactNode;
 }
 
+// Fix the overlapping icons/text when the error icon is shown.
+// XXX: once we've moved away from bootstrap/formio 'component library', this fix and
+// @emotion/css can be removed again.
+const PAD_ERROR_ICON = css`
+  .form-control.is-invalid {
+    padding-inline-end: calc(1.5em + 0.75rem);
+  }
+`;
```

Comment:
This restore some built in bootstrap CSS that the formio stylesheet manages to destroy :slightly_smiling_face: